src/transport/SessionManager.h

```diff
@@ -158,12 +158,13 @@ class DLL_EXPORT SessionManager : public TransportMgrDelegate
     /**
      * @brief
      *   Allocate a secure session and non-colliding session ID in the secure
-     *   session table.
+     *   session table. If known, the scoped nodeid of the peer that should be considered can be passed in if necessary
+     *   as part of the session allocation algorithm.
```

Comment:
I am getting lost in the "if known ... if necessary" qualifiers.

When should this argument be passed?  What should it contain?  How does one indicate "unknown"?